experimental/casemapping/src/exceptions_builder.rs

```diff
@@ -0,0 +1,205 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::error::Error;
+use crate::exceptions::{CaseMappingExceptions, ExceptionHeader, ExceptionSlot};
+use std::collections::HashMap;
+use zerovec::{VarZeroVec, ZeroVec};
+
+// CaseMappingExceptionsBuilder consumes the exceptions data produced by
```

Comment:
suggestion: add a comment about usage pattern for motivating use case

IIRC, this is intended to be used prior to runtime, when the data is being transformed. That contrasts with `UnicodeSetBuilder`, which is intended as a runtime thing. It might be good to document, since most `*Builder`s I've seen follow the fluent builder pattern for runtime user APIs.

Reply:
Will fix in future patch.